www-servers/lighttpd/lighttpd-1.4.39.recipe

```diff
@@ -14,7 +14,7 @@ SOURCE_URI="https://download.lighttpd.net/lighttpd/releases-1.4.x/lighttpd-$port
 CHECKSUM_SHA256="bc5f910f78b8cbae3e8fe1a1d8558259f09dbe50dd3907db771404bfab27bb2e"
 PATCHES="lighttpd-$portVersion.patch"
 
-ARCHITECTURES="x86_gcc2 ?x86 ?x86_64 ?arm ?ppc"
+ARCHITECTURES="x86_gcc2 !x86 !x86_64 ?arm ?ppc"
```

Comment:
Add -lnetwork to the LDFLAGS. Ideally the configure script should check for it using AC_SEARCH_LIBS.